### PR TITLE
docs: add performance docs

### PR DESCRIPTION
**Pull request by pmalek-sumo**

### CODE REVIEW

File: docs/Performance.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+Events Per Second (EPS) achieved for a particular average message size and CPU usage.
+
+|                       | 100B | 512B  | 1KB  | 5KB  | 10KB |
+|:---------------------:|:----:|:-----:|:----:|:----:|:----:|
+| **Average CPU usage** |  -   |   -   |  -   |  -   |  -   |
+|          5%           | 2000 | 1100  | 1000 | 150  | 200  |
+|          10%          | 3500 | 2100  | 1500 | 450  | 300  |
+|          20%          | 6500 | 4100  | 3000 | 1200 | 700  |
+|          50%          | 1400 | 10100 | 8500 |  -*  |  -*  |
+|          90%          |  -*  | 19100 |  -*  |  -*  |  -*  |
```
Comment (sumo-drosiek) [inline docs/Performance.md:28]: | **Average CPU usage** \ **Average Message Size**                      | 100B | 512B  | 1KB  | 5KB  | 10KB |
|:---------------------:|:----:|:-----:|:----:|:----:|:----:|
|          5%           | 2000 | 1100  | 1000 | 150  | 200  |
|          10%          | 3500 | 2100  | 1500 | 450  | 300  |
|          20%          | 6500 | 4100  | 3000 | 1200 | 700  |
|          50%          | 1400 | 10100 | 8500 |  -*  |  -*  |
|          90%          |  -*  | 19100 |  -*  |  -*  |  -*  |

vs


|                       | 100B | 512B  | 1KB  | 5KB  | 10KB |
|:---------------------:|:----:|:-----:|:----:|:----:|:----:|
| **Average CPU usage** |  -   |   -   |  -   |  -   |  -   |
|          5%           | 2000 | 1100  | 1000 | 150  | 200  |
|          10%          | 3500 | 2100  | 1500 | 450  | 300  |
|          20%          | 6500 | 4100  | 3000 | 1200 | 700  |
|          50%          | 1400 | 10100 | 8500 |  -*  |  -*  |
|          90%          |  -*  | 19100 |  -*  |  -*  |  -*  |

vs


|                       | **Average Message Size**  | 100B | 512B  | 1KB  | 5KB  | 10KB |
|:---------------------:|:----:|:-----:|:----:|:----:|:----:|:----:|
| **Average CPU usage** |  -   |   -   |  -   |  -   |  -   | - |
|          5%           |   -   | 2000 | 1100  | 1000 | 150  | 200  |
|          10%          |   -   | 3500 | 2100  | 1500 | 450  | 300  |
|          20%          |   -   | 6500 | 4100  | 3000 | 1200 | 700  |
|          50%          |   -   | 1400 | 10100 | 8500 |  -*  |  -*  |
|          90%          |    -   | -*  | 19100 |  -*  |  -*  |  -*  |

I don't really like any of them but I think first is the most correct

Comment (sumo-drosiek) [reply docs/Performance.md:28]: in addition `Average CPU usage` should be bolded

File: docs/Performance.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+Events Per Second (EPS) achieved for a particular average message size and CPU usage.
+
+|                       | 100B | 512B  | 1KB  | 5KB  | 10KB |
+|:---------------------:|:----:|:-----:|:----:|:----:|:----:|
+| **Average CPU usage** |  -   |   -   |  -   |  -   |  -   |
+|          5%           | 2000 | 1100  | 1000 | 150  | 200  |
+|          10%          | 3500 | 2100  | 1500 | 450  | 300  |
+|          20%          | 6500 | 4100  | 3000 | 1200 | 700  |
+|          50%          | 1400 | 10100 | 8500 |  -*  |  -*  |
```
Comment (sumo-drosiek) [inline docs/Performance.md:27]: Is 1400 correct value. What is the reason the EPS is so low?

File: docs/Performance.md (hunk 1 above)
Comment (pmalek-sumo) [reply docs/Performance.md:28]: I'd also say that the first is the most correct but I'd vote for second (as is) in terms of readability.

File: docs/Performance.md (hunk 2 above)
Comment (pmalek-sumo) [reply docs/Performance.md:27]: It's not, it should be 14000.

File: docs/Performance.md (hunk 1 above)
Comment (sumo-drosiek) [reply docs/Performance.md]: sadly agree with that

**Comment by pmalek-sumo** [comment]
Let's merge this as is and reword/restructure it when we have more results.

**Pull request merged**